hack/make/release-rpm

```diff
@@ -53,12 +54,12 @@ for distro in "${distros[@]}"; do
 		# sign the rpms before adding to repo
 		if [ ! -z $GPG_PASSPHRASE ]; then
 			# export our key to rpm import
-			gpg --armor --export releasedocker > /tmp/gpg
+			gpg --armor --export "$GPG_NAME_EMAIL" > /tmp/gpg
 			rpm --import /tmp/gpg
 
 			# sign the rpms
 			echo "yes" | setsid rpm \
-				--define '_gpg_name releasedocker' \
+				--define "_gpg_name $GPG_NAME_EMAIL" \
```

Comment:
for consistency if this one is `"` can we change them all, otherwise it looks weird or it can be like the `GPG_PASSPHRASE` line below